src/popup/components/TokenAmount.vue

```diff
@@ -48,6 +48,7 @@ export default {
     ...mapState({
       amountFiat(state, { convertToCurrency, formatCurrency }) {
         if (this.hideFiat || this.aex9) return '';
+        if (this.symbol.toUpperCase() !== 'AE') return '';
```

Comment:
TokenAmount has different props that can handle that. We shouldn't create another case for it